src/Interpreters/Streaming/SubstituteStreamingFunction.cpp

```diff
@@ -68,6 +69,51 @@ std::unordered_map<String, String> StreamingFunctionData::changelog_func_map = {
     {"moving_sum", ""},
 };
 
+std::pair<bool, String> StreamingFunctionData::supportChangelog(const String & function_name)
+{
+    auto iter = changelog_func_map.find(function_name);
+
+    /// Support combinator suffix, for example:
+    /// `count`                 => `__count_retract`
+    /// `count_if`              => `__count_retract_if`
+    /// `count_distinct`        => `__count_retract_distinct`
+    /// `count_distinct_if`     => `__count_retract_distinct_if`
+    String combinator_suffix;
+    auto nested_func_name = function_name;
+    while (iter == changelog_func_map.end())
+    {
+        if (auto combinator = AggregateFunctionCombinatorFactory::instance().tryFindSuffix(nested_func_name))
+        {
+            const std::string & combinator_name = combinator->getName();
```

Comment:
Since `getName` return a String value, there is no need to use a string reference here.